Cannot install CLI using Go v1.12 on a Mac

```
18:26 $ go get -u github.com/uber/astro/astro/cli/astro
# github.com/uber/astro/astro
../github.com/uber/astro/astro/sessions.go:176:14: cannot use func literal (type func(dag.Vertex) error) as type dag.WalkFunc in argument to graph.Walk
18:26 $ go version
go version go1.12 darwin/amd64
```

--- a/README.md
+++ b/README.md
@@ -15,13 +15,15 @@ NOTE: Astro is currently experimental.
 
 **Installation**
 
-Install Astro using go get (Go >1.11 required):
+Install Astro using go get (Go >1.12 required):
 
 ```
-go get -u github.com/uber/astro/astro/cli/astro
+GO111MODULE=on go get github.com/uber/astro/astro/cli/astro
 ```
 
 This will install a binary called `astro` in your `$GOPATH/bin`.
+
+Alternatively, you can download precompiled binaries from the [Github releases page](https://github.com/uber/astro/releases).
 
 **Configuration**
 

